src/connectedk8s/azext_connectedk8s/custom.py

```diff
@@ -741,6 +743,51 @@ def delete_connectedk8s(cmd, client, resource_group_name, cluster_name,
     # Check Release Existance
     release_namespace = get_release_namespace(kube_config, kube_context, helm_client_location)
 
+    # Check forced delete flag
+    if(force_delete):
+
+        confirmation_message = "You can check using 'kubectl config get-contexts' to check if your current context is pointing to the the right cluster. \n" + "Are you sure you want to execute the delete command:"
+        utils.user_confirmation(confirmation_message, False)
+
+        kubectl_client_location = install_kubectl_client()
+
+        delete_cc_resource(client, resource_group_name, cluster_name, no_wait).result()
+
+        # Explicit CRD Deletion
+        for crds in consts.Connected_Cluster_CRDs:
+            cmd_helm_delete = [kubectl_client_location, "delete", "crds", crds, "--ignore-not-found"]
+            response_helm_delete = Popen(cmd_helm_delete, stdout=PIPE, stderr=PIPE)
+            _, error_helm_delete = response_helm_delete.communicate()
+
+        # Timer added to have sufficient time after CRD deletion
+        # to check the status of the CRD ( deleted or terminating )
+        time.sleep(1)
```

Comment:
Why not use the GET request to query and confirm the status, but only sleep for a fixed one second?

Reply:
We saw the case when the CRD doesn't get deleted even after explicitly running the delete command because of the finalizer entering into a deadlock state because of operators getting deleted
Each CRD itself takes around 1/3 second to delete, so we added an additional wait timer for to give each of the CRD sufficient time to either delete and if not deleted, to change its status into "terminating"

Reply:
Got it, thanks~